admin/client/src/styles/legacy/_style.scss

```diff
@@ -1163,13 +1163,13 @@ body.cms {
 }
 
 .cms-tree-filtered {
-  position: absolute;
   margin: 0;
-  width: 100%;
   box-sizing: border-box;
-  margin-left: -$grid-x*2;
-  padding: $grid-y*2 $grid-x*2;
-  background: #D4E2EC;
+  margin-left: -$panel-padding-x;
+  margin-right: -$panel-padding-x;
+  height: calc(100% + #{$panel-padding-y});
```

Comment:
This leaves a lot of space below the last page item, and for some reason gives a "snapping"/"jumping" experience when I scroll to the bottom.
I'd suggest a `margin-bottom: -$panel-padding-y;` to give a more consistent experience.
![screen shot 2016-10-25 at 10 55 16 am](https://cloud.githubusercontent.com/assets/1064889/19665543/f0fe4682-9aa1-11e6-99b5-e4b12a1baa3e.png)